core/internal/cltest/client.go

```diff
@@ -123,7 +124,7 @@ func (c *SimulatedBackendClient) GetLogs(q ethereum.FilterQuery) (logs []eth.Log
 
 // SubscribeToLogs registers a subscription for push notifications of logs
 // from a given address.
-func (c *SimulatedBackendClient) SubscribeToLogs(ctx context.Context, channel chan<- eth.Log,
+func (c *SimulatedBackendClient) SubscribeToLogs(ctx context.Context, channel chan<- models.Log,
```

Comment:
I wonder if we can namespace this further... so perhaps it's like `ethModels` or something?

Bit confusing that some of our models are back by database rows and some are just in-memory structs.

Reply:
I agree, but I think it's better suited to the already-planned followup PR.  Trying to keep scope limited here so that we can move forward ASAP with the other Eth refactor tasks.